libcontainer/specconv/spec_linux.go

```diff
@@ -178,18 +183,17 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 	config := &configs.Config{
 		Rootfs:       rootfsPath,
 		NoPivotRoot:  opts.NoPivotRoot,
-		Readonlyfs:   spec.Root.Readonly,
+		Readonlyfs:   root.Readonly,
 		Hostname:     spec.Hostname,
 		Labels:       append(labels, fmt.Sprintf("bundle=%s", cwd)),
 		NoNewKeyring: opts.NoNewKeyring,
 		Rootless:     opts.Rootless,
 	}
 
-	exists := false
 	for _, m := range spec.Mounts {
 		config.Mounts = append(config.Mounts, createLibcontainerMount(cwd, m))
 	}
-	if err := createDevices(spec, config); err != nil {
+	if err = createDevices(spec, config); err != nil {
```

Comment:
this change shouldn't matter

Reply:
right just a spurious annoying lint warning, works either way